src/app/blog/page.tsx

```
@@ -1,51 +1,54 @@
+					tag: artigo.tag,
+					titulo: artigo.titulo,
+					texto: artigo.texto.raw.children[0].children[0].text,
+					autor: artigo.autor,
+					avatar: artigo?.avatar?.url,
+					data: artigo.createdAt,
+					imagem: artigo?.capa?.url,
+				};
+			})
+		: [];
+	const artigoDestaque = artigos.sort(
+		(a: any, b: any) => new Date(b.data).getTime() - new Date(a.data).getTime(),
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)